src/app/shared/models/user.model.ts

```diff
@@ -8,4 +8,13 @@ export interface User {
   email?: string;
   phoneNumber?: string;
   role: string;
-}
\ No newline at end of file
+
+  constructor(info, id?: string) {
+    this.id = info.id;
```

Comment:
you put id twice
is there any reason to pass id into constructor?

Reply:
yeah, otherwise we cannot update the user. it is required by backend. that's why I deleted the interface and created class

Reply:
and I don't see the reason to add the id to the Reactive Form group. ofc I can pass this body directly to the service without Object creation, but I've decided to follow the standard flow as for other items (workshop, provider)